### PR TITLE
version 1.0.0: decode user, get token, tests, coverage

### PR DESCRIPTION
**Pull request by scarney81**
@lanetix/engineers - code review. reusable token store. this is not specific to lanetix-specific

Provides:
- decoded user object
- access to token
- override-able key to access token

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  "description": "React JTW store",
+  "license": "MIT",
+  "author": "Seth Carney",
+  "keywords": [
+    "react",
+    "jwt"
+  ],
+  "main": "index.js",
+  "scripts": {
+    "run-coverage": "istanbul cover --include-all-sources _mocha -- --compilers js:babel/register",
```
Comment (apechimp) [inline package.json]: This may not work. If it doesn't, use [this](https://github.com/lanetix/salesforce-sync/blob/master/package.json#L11) as an example

Comment (scarney81) [inline package.json]: This works. I've verified by actually breaking coverage.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    "lodash": "^3.9.3",
+    "mocha": "^2.2.5",
+    "sinon": "^1.15.3"
+  },
+  "dependencies": {
+    "cookie-monster": "0.0.7",
+    "jwt-decode": "^1.1.1",
+    "xtend": "^4.0.0"
+  }
+}
```
Comment (apechimp) [inline package.json]: I don't see a `transforms`which I think you'll need for `babelify` and fat arrows.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "react-jwt-store",
+  "version": "1.0.0",
+  "description": "React JTW store",
```
Comment (apechimp) [inline package.json]: `JWT`

File: package.json (hunk 2 above)
Comment (scarney81) [inline package.json]: Do you mean this? https://github.com/lanetix/react-jwt-store/commit/28df824a47df7209633866d41247a9a0e800385c

File: package.json (hunk 3 above)
Comment (scarney81) [inline package.json]: :+1:

File: package.json (hunk 1 above)
Comment (apechimp) [inline package.json]: Huh! I wonder what I was doing wrong that made me need that.

**Comment by apechimp** [comment]
LGTM :dancers:

**Pull request merged**